ui/admin/setup-edit.php

```diff
@@ -168,7 +168,7 @@
                     ),
                     'class' => 'pods-validate pods-validate-required'
                 ) ); ?>
-                    <input type="button" class="save-button button" value="OK" /> <a class="cancel" href="#cancel-edit">Cancel</a>
+                    <input type="button" class="save-button button" value="<?php _e( 'OK', 'pods' ); ?>" /> <a class="cancel" href="#cancel-edit"><?php _e( 'Cancel', 'pods' ); ?></a>
```

Comment:
Shouldn't this be something more like this?
```
<input type="button" class="save-button button" value="OK"><?php _e( 'OK', 'pods' ); ?></button>
```
I didn't see anything depending on the value being "OK", but it's possible that there is other code using that value.